src/org/folio/rest_v2/Eureka.groovy

```diff
@@ -0,0 +1,82 @@
+package org.folio.rest_v2
+
+import org.folio.models.OkapiTenant
+import org.folio.utilities.RequestException
+
+class Eureka extends Authorization {
+
+  Eureka(Object context, String okapiDomain, boolean debug = false) {
+    super(context, okapiDomain, debug)
+  }
+
+  boolean isApplicationRegistered(String applicationId) {
+
+    String url = generateKongUrl("/applications/${applicationId}")
+
+    try {
+      restClient.get(url).body
+      logger.info("Application ${applicationId} is already registered.")
+      return true
+    } catch (RequestException e) {
+      if (e.statusCode == HttpURLConnection.HTTP_NOT_FOUND) {
+        logger.info("Application id ${applicationId} not found in Application manager. Proceeding with registration.")
+        return false
+      } else {
+        throw new RequestException("Application manager is unavailable", e.statusCode)
+      }
+    }
+  }
+
+  def getDescriptorsList(applicationId) {
+
+    String bucketName = 'eureka-application-registry'
```

Comment:
@DmytrMoroz 

_**Enhancement:**_

It would be better to put this Bucket Name into [Constants.groovy](https://github.com/folio-org/pipelines-shared-library/blob/81101d54c4fa2aa6e1cf56c7d3a96905de40e261/src/org/folio/Constants.groovy) file.